pythonforandroid/bootstraps/common/build/whitelist.txt

```diff
@@ -1 +0,0 @@
-# put files here that you need to un-blacklist
```

Comment:
Is this an unintended deletion?

Reply:
oops, yes it seems so, should I undo that or are you doing it during merge?

Reply:
Please do undo it

Reply:
done